src/main/java/org/araymond/joal/core/torrent/watcher/TorrentFileWatcher.java

```diff
@@ -23,8 +24,7 @@
  */
 @Slf4j
 class TorrentFileWatcher {
-    // TODO : get back to a bigger value as soon as https://issues.apache.org/jira/browse/IO-535 is fixed
-    private static final int DEFAULT_SCAN_INTERVAL_MS = 2 * 1000;
+    private static final long DEFAULT_SCAN_INTERVAL_MS = SECONDS.toMillis(5);
```

Comment:
Hum not sure we understood this the same.

To mitigate the bug I had to lower the folder scan interval to prevent very long shutdown time.
And i was saying that instead of scanning the while folder every 2s (which is a waste of resource IMO), we could get back to 5 or 10s interval when the bug got resolved.

5ms is way to low. I'd go for 5s or 10s interval. Maybe 2s is file afte rall

Reply:
`SECONDS.toMillis(5)` -> `5000`

Reply:
🤣 my bad